README.md

```diff
@@ -1,77 +1,95 @@
 
 # NOMAD's Analysis plugin
-This is a plugin for [NOMAD](https://nomad-lab.eu) to facilitate analysis of processed entry archives using Jupyter notebooks.
+This is a plugin for [NOMAD](https://nomad-lab.eu) to facilitate analysis of processed entry archives using classes and functions which can then be import for using in Jupyter notebooks.
+
+
+<!-- MOVE THIS TO THE DOCUMENTATION PAGE OF THIS PLUGIN --->
 
 ## Getting started
 
 
 ### Install the dependencies
 
-You should create a virtual environment. You will need the `nomad-lab` package (and `pytest`).
-You need Python 3.9.
-
+Clone the project and in the workspace folder, create a virtual environment (note this project uses Python 3.9):
 ```sh
+git clone https://github.com/FAIRmat-NFDI/nomad-analysis.git
+cd nomad-analysis
 python3.9 -m venv .pyenv
-source .pyenv/bin/activate
+. .pyenv/bin/activate
+```
+
+Install the `nomad-lab` package:
+```sh
 pip install --upgrade pip
 pip install '.[dev]' --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
 ```
 
 **Note!**
 Until we have an official pypi NOMAD release with the plugins functionality. Make
-sure to include NOMAD's internal package registry (e.g. via `--index-url`).
+sure to include NOMAD's internal package registry (via `--index-url` in the above command).
 
 
 ### Run the tests
 
-You can run automated tests with `pytest`:
+You can the unit testing using the `pytest` package:
 
 ```sh
-pytest -svx tests
+python -m pytest -sv
+```
+
+We recommend to install the `coverage` and `coveralls` packages for a more comprehensive output of the testing:
+```sh
+pip install coverage coveralls
+python -m coverage run -m pytest -sv
 ```
 
 ## Development
 
-The plugin is still under development. If you would like to contribute, install the package in editable mode with the development dependencies:
+The plugin is still under development. If you would like to contribute, install the package in editable mode (with the added `-e` flag) with the development dependencies:
 
 ```sh
 pip install -e .[dev] --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
 ```
 
+
 ### Setting up plugin on your local installation
 Read the [NOMAD plugin documentation](https://nomad-lab.eu/prod/v1/staging/docs/howto/oasis/plugins_install.html) for all details on how to deploy the plugin on your NOMAD instance.
 
-You need to modify the ```nomad.yaml``` configuration file of your NOMAD instance.
-To include the analysis plugin you need to add the following lines: .
+You need to modify the ```src/nomad_analysis/nomad_plugin.yaml``` to define the plugin adding the following content:
+```yaml
+plugin_type: schema
+name: schemas/nomad_analysis
+description: |
+  This plugin is used to analyze parsed raw data for spectral profiles in the standard NOMAD schema.
+```
```

Comment:
As this content is already in `src/nomad_analysis/nomad_plugin.yaml`, do we need this in README?

Reply:
Well, being a fork from the parser example, I would keep it. But yes, I admit it is too much info here.

Reply:
Ok, check it out. I think it is more clear that the content is already there.

Reply:
Looks good to me! 
Please make the description same as the one in the `nomad_plugin.yaml` file.
> This plugin is used to analyze parsed raw data from the NOMAD standard schema.